Px.Utils/Language/MultilanguageStringExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace PxUtils.Language
+{
+    public static class MultilanguageStringExtensions
+    {
+        /// <summary>
+        /// Retrieves the unique value from a multilanguage string.
+        /// This method assumes that all language versions of the string are identical.
+        /// If the values differ, an exception is thrown.
+        /// </summary>
+        public static string UniqueValue(this MultilanguageString multilanguageString)
```

Comment:
UniqueValue name here is a bit confusing to me. UniqueValue gives me the expectation that the value would be unique compared to other multilanguage strings, which this method doesn't assure. As its purpose is to validate and return an identical or uniform value between different languages, maybe "UniformValue", "IdenticalValue" or something like that would be more descriptive?

Reply:
Renamed to UniformValue()